pkgs/development/python-modules/APScheduler/default.nix

```diff
@@ -45,15 +44,15 @@ buildPythonPackage rec {
     twisted
     mock
     gevent
-  ] ++ lib.optionals (!isPy3k) [ trollius ];
+  ];
 
   propagatedBuildInputs = [
     six
     pytz
     tzlocal
     funcsigs
     setuptools
-  ] ++ lib.optional (!isPy3k) futures;
+  ];
 
   disabledTests = lib.optionals stdenv.isDarwin [
     "test_submit_job"
```

Comment:
Maybe you could add yourself as a maintainer?

Reply:
No, this was just a random package bump.